Firestore/core/src/firebase/firestore/remote/connectivity_monitor.h

```diff
@@ -71,6 +71,9 @@ class ConnectivityMonitor {
   // Invokes callbacks only if the status changed.
   void MaybeInvokeCallbacks(NetworkStatus new_status);
 
+  // Invokes callbacks and set net status to `new_status`.
```

Comment:
nit: change "set net status" to "sets the new status"

Reply:
Done.